src/collections/_documentation/platforms/minidump/index.md

```diff
@@ -13,12 +13,14 @@ In order to receive symbolicated stack traces, you have to upload debug informat
 <!-- WIZARD -->
 ## Platform and Language Support
 
-Minidumps are currently supported for **Windows, macOS and Linux**. There is no limitation as to which programming language can be used. Sentry can also demangle symbols from the following languages; other languages will show the mangled name instead:
+Minidumps are currently supported for **Windows, macOS and Linux**. There is no limitation as to which programming language can be used. Sentry can also demangle symbols from the following languages:
 
 -   C and C++
 -   ObjectiveC and ObjectiveC++
 -   Swift
 -   Rust
+
+Other languages will show the mangled name instead.
```

Comment:
```suggestion
These other languages will show the mangled name instead.
```

Reply:
Actually it is saying languages *not* in the list above

Reply:
Aha. Okay, made a new suggestion.